Cargo.toml

```diff
@@ -76,3 +77,9 @@ raw_value = []
 # overflow the stack after deserialization has completed, including, but not
 # limited to, Display and Debug and Drop impls.
 unbounded_depth = []
+
+# Support alternate encoding modes for bytes. Available on Rust 1.40+
```

Comment:
Version requirement due to https://github.com/serde-rs/json/pull/656/files#diff-b4aea3e418ccdb71239b96952d9cddb6R470